src/ClusterServerPool.cpp

```diff
@@ -92,7 +92,13 @@ void ClusterServerPool::handleResponse(Handler* h, ConnectConnection* s, Request
     p.set(res->body());
     for (auto serv : mServPool) {
         serv->setUpdating(true);
+        if (serv->loading()) {
```

Comment:
What is the idea behind adding it inside the clusternode forever loop instead of doing this in a separate loop?

Reply:
basically, this block handles the response for cluster nodes executed every 5 sec. there was a loop already here which is traversing all the servers so just put a check here if the serv was in loading state or not and if it then makes a ping call.

Reply:
Not important then.